Reply/app/src/main/java/com/materialstudies/reply/util/BindingAdapter.kt

```diff
@@ -37,12 +37,13 @@ import com.bumptech.glide.request.RequestListener
 import com.bumptech.glide.request.target.Target
 import com.google.android.material.chip.Chip
 import com.google.android.material.elevation.ElevationOverlayProvider
+import com.google.android.material.elevation.SurfaceColors
 import com.materialstudies.reply.R
 
 @BindingAdapter(
-    "popupElevationOverlay"
+    "popupSurfaceColorElevation"
 )
-fun Spinner.bindPopupElevationOverlay(popupElevationOverlay: Float) {
+fun Spinner.popupSurfaceColorElevation(popupElevationOverlay: Float) {
```

Comment:
let's change the `popupElevationOverlay` param to `elevation` like your other adapter

Reply:
yeah, missed that and the bind.